build/forklift-api/Containerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.11-9 AS builder
+WORKDIR /app
+COPY --chown=1001:0 ./ ./
+ENV GOFLAGS "-mod=vendor -tags=strictfipsruntime"
+ENV GOEXPERIMENT strictfipsruntime
+RUN GOOS=linux GOARCH=amd64 go build -buildvcs=false -ldflags="-w -s" -o forklift-api github.com/konveyor/forklift-controller/pkg/forklift-api
+
+FROM registry.access.redhat.com/ubi9-minimal:9.4-1227.1725849298
+# Required to be able to get files from within the pod
+RUN microdnf -y install tar && microdnf clean all
+
+COPY --from=builder /app/forklift-api /usr/local/bin/forklift-api
+ENTRYPOINT ["/usr/local/bin/forklift-api"]
+LABEL \
+        com.redhat.component="forklift-api-container" \
+        name="forklift/forklift-api-rhel9" \
+        io.k8s.display-name="Forklift" \
+        io.k8s.description="Forklift API server is running Kubernetes validation and mutation webhooks for Forklift resources." \
+        io.openshift.tags="migration,forklift,mtv" \
+        summary="Forklift- API" \
+        description="Forklift - API" \
+        maintainer="Migration Toolkit for Virtualization Team <migtoolkit-virt@redhat.com>"
```

Comment:
@ShakedAviv50313 please change for `"Forklift by Konveyor Community <forklift-dev@googlegroups.com>"`